3 - PB/Documentazione esterna/Verbali/24-03-22/24-03-22.typ

```diff
@@ -0,0 +1,60 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "22/03/24",
+  subTitle: "Aggiornamento sullo stato dei lavori",
+  docType: "verbale",
+  authors: (
+    "Banzato Alessio",
+  ),
+  reviewers: (
+    "Gardin Giovanni",
+  ),
+  missingMembers: (
+    "Todesco Mattia",
+  ),
+  externalParticipants : (
+    (name: "Bassani Matteo", role: "Referente aziendale"),
+    (name: "De Sio Domenico", role: "Referente aziendale"),
+  ),
+  timeStart: "16:00",
+  timeEnd: "16:30",
+  location: "Zoom",
+);
+
+= Ordine del giorno
+- Presentazione MVP;
+- Presentazione dei pattern per la ricerca;
+- Lista dei bug;
+- Pianificazione meeting futuri.
+
+== Presentazione MVP
+
+Sono state presentate ai Proponenti le funzionalità dell'MVP sviluppate dall'ultimo incontro, in particolare:
+
+- configurazione dell'ambiente a partire da file SVG, e in conseguenza di questo:
+  - caricamento da database e posizionamento nell'ambiente degli scaffali;
+  - ridimensionamento del piano.
+- spostamento zone tramite _drag and drop_;
+- spostamento prodotti tra bin tramite _drag and drop_;
+- ispezione bin tramite doppio click;
+- ispezione zone tramite doppio click.
+
+I Proponenti sono risultati soddisfatti delle funzionalità implementate.
+
+Sono state inoltre illustrate le funzionalità che verranno implementate entro il prossimo incontro:
+
+- *possibilità di muoversi all'interno dell'ambiente tramite tastiera*: se l'utente vuole spostare un bin in una zona non presente nell'inquadratura, potrà muovere la camera utilizzando la tastiera, mantenendo quindi il prodotto ancorato al cursore. I Proponenti approvano questa scelta;
```

Comment:
In realtà questo è già stato parzialmente mostrato, quello che intendiamo implementare entro il prossimo sprint è la seconda parte, ossia muovere la cam durante il dragging dell'oggetto (per ora la camera si muove già con la tastiera ma non mentre si fa il dragging)